index.html

```diff
@@ -199,97 +196,107 @@ <h4 class="skills-item">javascript</h4>
-                    <div class="frameworks-container">
-                        <h3 class="languages-title">Frameworks</h3>
-                        <a href=""><img src="img/next-arrow.svg" alt="arrow icon"></a>
-                    </div>
-                </li>
-                <li class="about-li-category">
-                    <!-- sills -->
-                    <div class="frameworks-container">
-                        <h3 class="languages-title">Skills</h3>
-                        <a href=""><img src="img/next-arrow.svg" alt="arrow icon"></a>
-                    </div>
-                </li>
+                    <li class="about-li-category">
+                        <!-- sills -->
+                        <div class="frameworks-container">
+                            <h3 class="languages-title">Skills</h3>
+                            <a href=""><img src="img/next-arrow.svg" alt="arrow icon"></a>
+                        </div>
+                    </li>
+                </div>
 
             </section>
             <section class="contact-section">
-                <h4 class="contact">Contact me</h4>
-                <p class="contact-text">If you have an application you are interested in developing, a feature that you
-                    need built or a project that needs coding. I’d love to help with it </p>
-                <form action="https://formspree.io/f/mbjbpybo" method="post" class="contact-form">
-                    <input placeholder="name" class="name" maxlength="30" name="name" type="text" required>
-                    <input placeholder="email" class="email" name="email" type="email" required>
-                    <textarea name="message" id="message" cols="30" rows="10" maxlength="500" class="text-box"
-                        placeholder="write your message here" type="text" required></textarea>
-                    <button class="button-contact" type="submit">Get in touch </button>
-                </form>
+                <div>
+                    <h4 class="contact">Contact me</h4>
+                    <p class="contact-text">If you have an application you are interested in developing, a feature that
+                        you
+                        need built or a project that needs coding. I’d love to help with it </p>
+                    <form action="https://formspree.io/f/mbjbpybo" method="post" class="contact-form">
+                        <input placeholder="name" class="name" maxlength="30" name="name" type="text" required>
+                        <input placeholder="email" class="email" name="email" type="email" required>
+                        <textarea name="message" id="message" cols="30" rows="10" maxlength="500" class="text-box"
+                            placeholder="write your message here" type="text" required></textarea>
+                        <button class="button-contact" type="submit">Get in touch </button>
+                    </form>
+                </div>
+
```

Comment:
[OPTIONAL]Kindly fix the width of the contact form to match the figma design
your project
<img width="1673" alt="Screenshot 2022-11-03 at 22 04 12" src="https://user-images.githubusercontent.com/48546992/199814251-3e5f9df5-163e-45d1-ae11-2ecc0392e92a.png">

Figma design:
<img width="712" alt="Screenshot 2022-11-03 at 22 04 23" src="https://user-images.githubusercontent.com/48546992/199814310-b7918db9-d20e-4cd5-8fbb-bb5ae255f566.png">

Reply:
thank you ! let me update it quickly !
I really appreciate 🤝